docs/design-docs/01-oci-support.md

```diff
@@ -0,0 +1,338 @@
+# OCI Support
+
+* Author(s): Louis Dejardin, @loudej
+* Approver: \<kpt-maintainer\>
+
+>    Every feature will need design sign off an PR approval from a core
+>    maintainer.  If you have not got in touch with anyone yet, you can leave
+>    this blank and we will try to line someone up for you.
+
+## Why
+
+Systems that deal with packaging software or bundling configuration often have
+an atomic, versionable artifact. This artifact can exist as a source of
+truth independant from the source controlled content from which it was built.
+
+It is also very common for those packages to have an associated feed or repository
+which can receive those packages as they are published, and make them available for 
+download as needed. In many companies, using `git` as the as the repository and source
+of truth for production configuration comes with challenges.
```

Comment:
What challenges ?

Reply:
Do we have any specific customers looking for this feature ? If so, what is their use-case and their e2e flow look like ?

Reply:
Challenges include securing git, geo-replication requirements, practical limits operating at scale, using git as a source of truth in air-gapped environments.

For all of those kinds of challenges, if a customer has solved how to securely deliver pod container images to cluster via registry, then (with OCI support in Config Sync and Kpt) it would be literally the same practices and registry services used to securely deliver configuration package images.

Reply:
Yes, for customers, ping Yoshi?